src/DynamoCore/BuiltInAndOperators/BuiltIn.Migrations.xml

```diff
@@ -72,6 +72,16 @@
       </attribute>
     </additionalAttributes>
   </priorNameHint>
+  <priorNameHint>
+    <oldName>RemoveNulls</oldName>
+    <newName>DSCore.List.Clean</newName>
```

Comment:
@aparajit-pratap I thought the migration framework will not do it's job after 2.0 but maybe our team dis the wrong estimation that the XML code will only used for parsing such file? As long as we keep this file's file format, does it mean we can still do node migration?

Reply:
I think as @alfarok has told us a couple times :) this should work after his most recent changes to the deserialization logic - what we are missing is the framework for doing more complex migrations.

Reply:
@mjkkirschner @alfarok What kind of more complex migrations are there? If this way is acceptable for Civil, we can still say nodes migration supported in a limited way, can we align within our team?

Reply:
@QilongTang migrations like this:
https://github.com/DynamoDS/Dynamo/pull/7582